f2/src/components/formik/alert/index.js

```diff
@@ -30,7 +29,7 @@ const Summary = styled(BaseAlert, {
 `
 
 const iconStyle = () =>
-  css`
+  `
```

Comment:
I know all CSS are hardcoded here.  And we don't need css helper function. But do they produce the same CSS?

Reply:
Yes, the resulting components are styled in the same way.